### PR TITLE
Hide disable extension setting

### PR DESCRIPTION
**Pull request by ebidel**
R: @brendankenny 

Didn't remove code or html for now so we can test if this setting is impactful.

### CODE REVIEW

File: lighthouse-extension/app/popup.html [hunk 1]
```diff
@@ -57,10 +57,10 @@ <h2 class="header-titles__url">...</h2>
   </aside>
 
   <aside class="options subpage">
-    <h2 class="options__title">Settings</h2>
-    <div>
+    <h2 class="options__title" hidden>Settings</h2>
+    <div hidden>
       <label>
-        <input type="checkbox" class="setting-disable-extensions"
+        <input type="checkbox" class="setting-disable-extensions" disabled
```
Comment (ebidel) [inline lighthouse-extension/app/popup.html]: for good measure

**Comment by brendankenny** [comment]
does more need to be disabled in `lighthouse-background.js`? e.g. the part that gets all the current extensions run with or without the checkbox checked. 

maybe comment out the part that reads/writes `disableExtensionsDuringRun` from storage? mostly for people that may have already enabled, but also to be double extra sure :)

**Comment by brendankenny** [comment]
LGTM and runs well. Do you want to file an issue to track testing of this?

**Comment by ebidel** [comment]
https://github.com/GoogleChrome/lighthouse/issues/1689

**Pull request merged**